### PR TITLE
Add an integration test for ambassador addon, fix filename and bump operator version

### PR DESCRIPTION
**Pull request by concaf**
This PR adds an integration test for the ambassador addon as discussed in #8161. Also bumps the operator version.

CC: @priyawadhwa @medyagh

### CODE REVIEW

File: deploy/addons/ambassador/ambassadorinstallation.yaml [hunk 1]
```diff
@@ -7,3 +7,5 @@ spec:
   installOSS: true
   helmValues:
     deploymentTool: amb-oper-minikube
+    deploymentStrategy:
+      type: Recreate
```
Comment (medyagh) [inline deploy/addons/ambassador/ambassadorinstallation.yaml:11]: is this newline needed ?

File: pkg/minikube/assets/addons.go [hunk 2]
```diff
@@ -407,7 +407,7 @@ var Addons = map[string]*Addon{
 		MustBinAsset(
 			"deploy/addons/ambassador/ambassadorinstallation.yaml",
 			vmpath.GuestAddonsDir,
-			"ambassadorinstallation.yaml.yaml",
+			"ambassadorinstallation.yaml",
```
Comment (medyagh) [inline pkg/minikube/assets/addons.go:410]: was this a bug that this is trying to fix ? if yes please change the PR title

**Comment by priyawadhwa** [comment]
Hey @concaf thank you for opening this PR. Are you still working on this? If so could you please:

1. Address review comments
1. Fix the integration tests you wrote; [TestAddons](https://storage.googleapis.com/minikube-builds/logs/8372/3d6b3b3/VirtualBox_Linux.html#fail_TestAddons) is failing across multiple testing environments

**Comment by concaf** [comment]
Hey @priyawadhwa, I'm winding up a few things right now, I'll be fixing this next week. Sorry.

**Comment by priyawadhwa** [comment]
No worries, totally understand. Just checking in :)

**Comment by concaf** [comment]
I've sent https://github.com/kubernetes/minikube/pull/8792 which fixes intermittent failures while deploying ambassador (and dashboard) addon with the `none` driver. Keeping an eye out for the rest of the tests in CI right now.

File: pkg/minikube/assets/addons.go (hunk 2 above)
Comment (concaf) [reply pkg/minikube/assets/addons.go:418]: updated

File: deploy/addons/ambassador/ambassadorinstallation.yaml (hunk 1 above)
Comment (concaf) [reply deploy/addons/ambassador/ambassadorinstallation.yaml]: Added a newline here :+1:

**Comment by concaf** [comment]
Hey @priyawadhwa @medyagh, I've addressed the review comments and fixed the tests. I don't see any failures on "none_Linux" and "KVM_Linux". Failures on "Docker_Linux" and "VirtualBox_Linux" look unrelated. Waiting on other CI results but I think this is ready for another shot ;)

**Comment by concaf** [comment]
Gentle nudge :wink:

**Comment by tstromberg** [comment]
Fixed merge conflict.

/ok-to-test

**Comment by medyagh** [comment]
@concaf sorry for the long delay in the PR review, do you mind pulling master one more time ? feel free to ping me on slack if got delayed again

**Comment by medyagh** [comment]
@concaf would you mind please rebasing the PR, otherwise it looks good

**Comment by medyagh** [comment]
@concaf do u mind please to solve merge conflicts and try one more time?

**Comment by tstromberg** [comment]
Please re-open once merge conflicts are addressed. Thanks!

**Comment by concaf** [comment]
Apologies for the delay - I've lost my dev setup to rebase and test this and I don't have the bandwidth to fix this right now.
CC: @kflynn @khussey do you folks have someone to do a quick rebase and test this - should not take long.

**Pull request closed without merging**